tasks/client.yaml

```diff
@@ -1,25 +1,154 @@
 ---
-- name: Operating system dependencies
-  apt: name={{ item }} state=present
+
+# dependencies
```

Comment:
That's exactly how it's done here: [tasks/main.yml#L9-L10](https://github.com/gronke/ansible-letsencrypt/blob/d0ccc8bb814af836a02097985ad368334c57f063/tasks/main.yml#L9-L10)